vllm/attention/backends/abstract.py

```diff
@@ -40,6 +40,10 @@ class AttentionBackend(ABC):
     # makes sure the output tensor is allocated inside the cudagraph.
     accept_output_buffer: bool = False
 
+    # Whether the attention backend can return the softmax lse for decode.
+    # Some features like decode context parallelism require the softmax lse.
+    can_return_lse_for_decode: bool = False
```

Comment:
I use the name `can_return_lse_for_decode`, because some attention backends might not always want to return lse even if they can return lse. They can compute `need_to_return_lse_for_decode = can_return_lse_for_decode and dcp_size > 1` and then decide if lse is required.